packages/gatsby-plugin-image/src/components/layout-wrapper.tsx

```diff
@@ -44,12 +44,12 @@ export const LayoutWrapper: FunctionComponent<ILayoutWrapperProps> = function La
   children,
 }) {
   let sizer: JSX.Element | null = null
-  if (layout === `responsive`) {
+  if (layout === `fluid`) {
     sizer = (
-      <div aria-hidden style={{ paddingTop: `${(width / height) * 100}%` }} />
+      <div aria-hidden style={{ paddingTop: `${(height / width) * 100}%` }} />
```

Comment:
Intentional?

Reply:
Yes, it was a very annoying bug to track down!

Reply:
Ok, great :)

If it was difficult to track down maybe it warrants its own PR, prior to this one, so you can explain what's going on and not drown in a 2000 line delta change.